homeassistant/components/tuya/config_flow.py

```diff
@@ -26,6 +26,8 @@
 RESULT_SINGLE_INSTANCE = "single_instance_allowed"
 RESULT_AUTH_FAILED = "invalid_auth"
 TUYA_ENDPOINT_BASE = "https://openapi.tuyacn.com"
+TUYA_ENDPOINT_OTHER = "https://openapi.tuyaus.com"
+COUNTRY_CODE_CHINA = ["86", "+86", "China"]
```

Comment:
```suggestion
COUNTRY_CODE_CHINA = "86"
```

According to the documentation, this is the only valid code.

Reply:
We add three array values to increase the fault tolerance, we don’t know what users will input. So will keep this code, no need to change it.

Reply:
Country code is also sent to the server. Does the server not care? https://github.com/tuya/tuya-iot-python-sdk/blob/master/tuya_iot/openapi.py#L208-L216

Reply:
The server will help handle the country code.